lib/rules-en.js

```diff
@@ -5,7 +5,7 @@ const normalize = require('./normalize');
 
 // These two lists contain all ASCII symbols, where symbol is defined as a
 // printable ASCII character not matching [ 0-9a-zA-z].
-const ASCII_PUNCTUATION_STR = `!"',-.:;?\``;
+const ASCII_PUNCTUATION_STR = `!"',-.:;?\`´`;
```

Comment:
The functionality makes a lot of sense to me, however, this character isn't in the ASCII character set, so this variable name and the comments around it would be inaccurate. I see a few lines below that we have COMMON_NONASCII_PUNCTUATION_STR -- could the same functionality be achieved by adding this character to it instead?

Reply:
Yep, I can move it there!

Reply:
Thanks!